app/controllers/tickets_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class TicketsController < ApplicationController
+  def show
+    respond_to do |format|
+      format.html do
+        @ticket_id = params.require(:id)
+        render :index
+      end
+      format.json do
+        ticket = Ticket.find(params.require(:id))
+        requester_stakeholders = ticket.user_stakeholders(current_user)
+
+        return head :unauthorized if requester_stakeholders.empty?
```

Comment:
Why? If there are no (active) stakeholders, why does that imply that the user doesn't have authorization to view the ticket?

Reply:
Okay, I understand the context now. Based on your explanation, shouldn't it be something like `return head :unauthorized unless requester_stakeholders.include?(current_user)`?

Reply:
Okay, makes more sense now. What about including this "WRT example" from your most recent comment as a code comment as well?

Reply:
Okay, I've added comments 2-3 lines above - before initializing `requester_stakeholders`. Hope it's fine.